index.html

```diff
@@ -105,92 +105,14 @@ <h3>Multi-Post Stories</h3>
-            <li><img class="langicon" src="images/ruby2.png" alt="ruby logo"></li>
-          </ul>
-        </div>
-        <button type="button" class="btn">See Project</button> 
-        
-        <div class="text1">
-          <h3 class="title-post">Professional Art Printing Data</h3>
-          <p>A daily selection of privately personalized reads; no accounts or
-          sign-ups required.has been the industry's standard
-          </p>
-          <ul class="icons">
-            <li><img class="langicon" src="images/html2.png" alt="html logo"></li>
-            <li><img class="langicon" src="images/bootstrap2.png" alt="bootstrap logo"></li>
-            <li><img class="langicon" src="images/ruby2.png" alt="ruby logo"></li>
-          </ul>
-        </div>
-        <button type="button" class="btn">See Project</button>
-
-        <div class="text1">
-          <h3 class="title-post">Professional Art Printing Data</h3>
-          <p>A daily selection of privately personalized reads; no accounts or
-          sign-ups required.has been the industry's standard
-          </p>
-          <ul class="icons">
-            <li><img class="langicon" src="images/html2.png" alt="html logo"></li>
-            <li><img class="langicon" src="images/bootstrap2.png" alt="bootstrap logo"></li>
-            <li><img class="langicon" src="images/ruby2.png" alt="ruby logo"></li>
-          </ul>
-        </div>
-        <button type="button" class="btn">See Project</button>
-        
-        <div class="text1">
-          <h3 class="title-post">Professional Art Printing Data</h3>
-          <p>A daily selection of privately personalized reads; no accounts or
-          sign-ups required.has been the industry's standard
-          </p>
-          <ul class="icons">
-            <li><img class="langicon" src="images/html2.png" alt="html logo"></li>
-            <li><img class="langicon" src="images/bootstrap2.png" alt="bootstrap logo"></li>
-            <li><img class="langicon" src="images/ruby2.png" alt="ruby logo"></li>
-          </ul>
-        </div>
-        <button type="button" class="btn">See Project</button>
-
-        <div class="text1">
-          <h3 class="title-post">Professional Art Printing Data</h3>
-          <p>A daily selection of privately personalized reads; no accounts or
-          sign-ups required.has been the industry's standard
-          </p>
-          <ul class="icons">
-            <li><img class="langicon" src="images/html2.png" alt="html logo"></li>
-            <li><img class="langicon" src="images/bootstrap2.png" alt="bootstrap logo"></li>
-            <li><img class="langicon" src="images/ruby2.png" alt="ruby logo"></li>
-          </ul>
-        </div>
-        <button type="button" class="btn">See Project</button>
+        <button type="button" class="btn-2">See Project</button>
       </section>
+      
+      <section class="projects" id="mobile-proj"></section>
```

Comment:
- [ ] Please the `projects section` is created dynamically with the information from the projects array. Remember the [Project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md): All of the HTML in that section is created when the page loads.
Therefore, What you need to do is:
- Kindly change the existing code; the already created HTML code for the `projects section` list should be removed, and instead it should be added via JavaScript